aggregator/src/test/scala/com/stratio/sparkta/aggregator/DataCubeSpec.scala

```diff
@@ -17,60 +17,80 @@
 package com.stratio.sparkta.aggregator
 
 import java.io.{Serializable => JSerializable}
-import scala.collection.mutable
-
-import org.apache.spark._
-import org.apache.spark.rdd.RDD
-import org.apache.spark.streaming._
-import org.junit.runner.RunWith
-import org.scalatest.FlatSpec
-import org.scalatest.junit.JUnitRunner
 
 import com.stratio.sparkta.plugin.bucketer.passthrough.PassthroughBucketer
 import com.stratio.sparkta.plugin.operator.count.CountOperator
 import com.stratio.sparkta.sdk._
+import org.apache.spark.streaming.TestSuiteBase
+import org.scalatest.{FeatureSpec, FlatSpec}
 
-@RunWith(classOf[JUnitRunner])
-class DataCubeSpec  extends FlatSpec{
-
-  "DataCubeSpec" should "process the events queued" in {
-
-    val myConf = new SparkConf()
-      .setAppName(this.getClass.getSimpleName + "" + System.currentTimeMillis())
-      .setMaster("local[2]").set("spark.streaming.clock", "org.apache.spark.streaming.util.ManualClock")
-    val sc = new SparkContext(myConf)
-    val ssc: StreamingContext = new StreamingContext(sc, Seconds(2))
-    val clock = new ClockWrapper(ssc)
-
-    def myDim(i: Int) = new Dimension("myKey" + i, new PassthroughBucketer().asInstanceOf[Bucketer])
-
-    def getDimensions: Seq[Dimension] = (0 until 10) map (i => myDim(i))
-
-    def getComponents: Seq[(Dimension, BucketType)] = (0 until 1) map (i => (myDim(i), Bucketer.identity))
-
-    def getOperators: Seq[Operator] = (0 until 1) map (i => new CountOperator(Map("prop" -> "propValue"
-      .asInstanceOf[JSerializable])))
-
-    val myRollups: Seq[Rollup] = (0 until 2) map (i => new Rollup(components = getComponents, operators = getOperators))
-
-    val events = mutable.Queue[RDD[Event]]()
-    val myDimensions: Seq[Dimension] = getDimensions
-
-    val getEvents: Seq[Event] = (0 until 10) map (i => new Event(Map("key" + i -> System.currentTimeMillis()
-      .asInstanceOf[JSerializable]), Some("myRaw event data here")))
-
-
-    val dc: DataCube = new DataCube(myDimensions, myRollups)
-    val result = dc.setUp(ssc.queueStream(events))
-    result.foreach(i => i.print)
-
-    assert(result.size == 2)
-
-    ssc.start()
-    events += sc.makeRDD(getEvents)
-    clock.advance(2)
-
-    ssc.stop()
-
+/**
+ * Tests over DataCube operations.
+ * @author anistal
+ */
+class DataCubeSpec extends TestSuiteBase {
```

Comment:
Is it really needed? DataCubeSpec is extending from TestSuiteBase and in turn extends from FanSuite. Scala should be run this test without use JUnitRunner.  Am I wrong?